--- libelysium-0.2.6/configure.in~	Tue Dec 11 11:58:33 2001
+++ libelysium-0.2.6/configure.in	Tue Dec 11 11:58:33 2001
@@ -1,8 +1,6 @@
 AC_INIT(configure.in)
 
-VERSION="0.2.6"
-
-AM_INIT_AUTOMAKE(libelysium, $VERSION)
+AM_INIT_AUTOMAKE(libelysium, 0.2.6)
 
 ELYSIUM_LIBDIR='-L${libdir}'
 ELYSIUM_INCLUDEDIR='-I${includedir}'
@@ -16,14 +14,12 @@
 AC_ARG_PROGRAM
 AM_PROG_LIBTOOL
 
-CFLAGS="`glib-config --cflags`"
-LDFLAGS="`glib-config --libs`"
+CFLAGS="$CFLAGS `glib-config --cflags`"
+LDFLAGS="$LDFLAGS `glib-config --libs`"
 
 AC_SUBST(ELYSIUM_LIBDIR)
 AC_SUBST(ELYSIUM_INCLUDEDIR)
 AC_SUBST(ELYSIUM_LIBS)
-AC_SUBST(CFLAGS)
-AC_SUBST(LDFLAGS)
 
 AC_OUTPUT([
 libelysium.spec
